d2common/d2data/d2datadict/object_group.go

```diff
@@ -0,0 +1,112 @@
+package d2datadict
+
+import (
+	"fmt"
+	"log"
+	"strconv"
+
+	"github.com/OpenDiablo2/OpenDiablo2/d2common"
+)
+
+const (
+	objectsGroupSize     = 7
+	memberDensityMin     = 0
+	memberDensityMax     = 125
+	memberProbabilityMin = 0
+	memberProbabilityMax = 100
+	expansionDataMarker  = "EXPANSION"
+)
+
+// ObjectGroupRecord represents a single line in objgroup.txt.
+// Information has been gathered from [https://d2mods.info/forum/kb/viewarticle?a=394].
+type ObjectGroupRecord struct {
+	// GroupName is the name of the group.
+	GroupName string
+
+	// Offset is the ID of the group, referred to by Levels.txt.
+	Offset int
+
+	// Members are the objects in the group.
+	Members *[objectsGroupSize]ObjectGroupMember
```

Comment:
Did you mean to put `Members [objectsGroupSize]*ObjectGroupMember` here ?

Reply:
I meant it as a pointer to a fixed-size array.

Reply:
Ok